docs/design/features/readytorun-elf-support-roadmap.md

```diff
@@ -0,0 +1,166 @@
+Roadmap to native Linux executable format (ELF) support in Ready to Run
+=======================================================================
+
+Revisions:
+* 1.1 - [Tomas Rylek](https://github.com/trylek) - 2/27/2020
+
+# Introduction
+
+This document details the steps needed for supporting native Linux executable
+format (ELF) for ready-to-run binaries on Linux. Today, ready-to-run binaries
```

Comment:
Since ELF is supported by non Linux-based as well, could it be kept neutral from that perspective, e.g.:
```suggestion
This document details the steps needed for supporting native executable and linkable
format (ELF) for ready-to-run binaries on Unix-like operating systems. Today, ready-to-run binaries
```
or something that implies `systems that support ELF` (e.g. macOS uses `MACH-O`).